Scaling Section Review

jarolrod opened the pull request:
Various fix-ups and re-wording throughout the scaling section. Changes broken up into separate commits for easy review.

Note to reviewer: please check render of the changes is fine, I did not test for this.

kallerosenbaum commented on scaling.adoc:
```diff
@@ -235,37 +234,36 @@ different versions of Bitcoin Core going back to version 0.8.
 
-There are also several good examples of where space savings have
-contributed to performance improvements. In a
+TThere are also several good examples of space savings contributing to
```
TT

jarolrod replied:
thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809

kallerosenbaum commented on scaling.adoc:
```diff
@@ -340,23 +338,23 @@ theme is that the participants create a number of semi-signed
 
 In a https://www.reddit.com/r/Bitcoin/comments/438hx0/a_trip_to_the_moon_requires_a_rocket_with/[Reddit post] titled "`A trip to the moon requires a rocket with
-multiple stages or otherwise the rocket equation will eat your
+multiple stages or otherwise, the rocket equation will eat your
```
This is a quote and should not be altered.

jarolrod replied:
thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809

jarolrod commented:
updated from 5bab192 to 0e8d06c ([pr28.01](https://github.com/jarolrod/btcphilosophy/tree/pr28.01) -> [pr28.02](https://github.com/jarolrod/btcphilosophy/tree/pr28.02), [diff](https://github.com/jarolrod/btcphilosophy/compare/pr28.01..pr28.02))

addressed @kallerosenbaum comments ([1](https://github.com/kallerosenbaum/btcphilosophy/pull/28#discussion_r874567014), [2](https://github.com/kallerosenbaum/btcphilosophy/pull/28#discussion_r874572178))
